spec/models/blog_post_spec.rb

```diff
@@ -44,6 +48,19 @@
       published2 = create(:blog_post, :published_at => Time.now)
       expect(BlogPost.all).to eq([published2, published1])
     end
+
+    context "language" do
```

Comment:
"language" doesn't say much for a context, it has no state, which is what a context is

"default locale" might